### PR TITLE
Update status and stage

### PR DESCRIPTION
**Pull request by pzuraq**

### CODE REVIEW

**Comment by matthewrobb** [comment]
I take it the meetings are going well!

**Comment by hax** [comment]
Hope we could have a small section for implementers and decorator authors to explain what's the remain issues and which part would be risk to change after stage 3.

File: README.md [hunk 1]
```diff
@@ -728,11 +728,9 @@ Decorators on further constructs are investigated in [EXTENSIONS.md](./EXTENSION
 - [x] Implement in experimental transpilers
   - STATUS: An experimental implementation has been created and is available for general use. Work is ongoing to implement in Babel and get more feedback.
     - [x] Independent implentation: https://javascriptdecorators.org/
```
Comment (kt3k) [inline README.md]:
```suggestion
    - [x] Independent implementation: https://javascriptdecorators.org/
```

File: README.md [hunk 2]
```diff
@@ -728,11 +728,9 @@ Decorators on further constructs are investigated in [EXTENSIONS.md](./EXTENSION
...
-    - [ ] Babel plugin implementation
-- [ ] Collect feedback from JavaScript developers testing the transpiler implementation
-  - STATUS: Feedback is being collected using the experimental implementation. More feedback will be collected from the Babel plugin once it is completed.
-- [ ] Propose for Stage 3 no sooner than six months after prototyping begins, so we have time to collect experience from developers in transpilers
-
+    - [x] Babel plugin implementation
```
Comment (nicolo-ribaudo) [inline README.md]:
```suggestion
    - [x] Babel plugin implementation ([docs](https://babeljs.io/docs/en/babel-plugin-proposal-decorators#options))
```

Just to save a google search for people reading this and wondering how to enable them in their project.

**Comment by pzuraq** [comment]
@hax I'll open a separate PR for that, but the open issues list should now be accurate for the remaining issues we have to solve in general.

**Pull request merged**